### ISSUE TITLE
windows-vcpkg test is flaky

### ISSUE THREAD

**Issue by Skepfyr** [sfackler/rust-openssl#1862]
The windows-vcpkg CI job is occasionally failing in the cargo-test phase with
> Caused by:
  process didn't exit successfully: `D:\a\rust-openssl\rust-openssl\target\debug\deps\openssl-2ac4bf41cbb50970.exe` (exit code: 0xc0000264)

Which, according to [NT Status Values](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/596a1078-e883-4972-9bbc-49e60bebca55), means: 

STATUS_RESOURCE_NOT_OWNED: {Application Error} The application attempted to release a resource it did not own. Click OK to terminate the application.

**Comment by sfackler** [comment]
This sounds suspiciously like https://github.com/openssl/openssl/issues/20613 manifesting as an error in Windows rather than a deadlock like in Linux.

**Comment by alex** [comment]
Yes, it did bump. This is fixed by skipping these tests. They can be unskipped when 3.1.1. is out
